packages/terra-clinical-label-value-view/src/LabelValueView.jsx

```diff
@@ -44,13 +44,28 @@ const LabelValueView = ({
   } else if (textValue) {
     textValueSection = <div className={cx('value')}>{textValue}</div>;
   }
+  const termDefinition = (
+    <React.Fragment>
+      <dt className={cx('label')}>{label}</dt>
+      <dd className={cx('value-wrapper')}>
+        {textValueSection}
+        {children}
+      </dd>
+    </React.Fragment>
+  );
+
+  if (customProps.childOfDescriptionList === 'true') {
```

Comment:
nit: can this be added as a regular prop like `label` or `value` rather than a custom prop? Just so that it shows up on the documentation site as consumers will need to set this.